src/SortableContainer/index.js

```diff
@@ -168,22 +163,17 @@ export default function SortableContainer(WrappedComponent, config = {withRef: f
 			const active = this.manager.getActive();
 
 			if (active) {
-				const {axis, onSortStart, helperClass, hideSortableGhost, useWindowAsScrollContainer, getHelperDimensions} = this.props;
+				const {axis, onSortStart, helperClass, hideSortableGhost, useWindowAsScrollContainer} = this.props;
 				let {node, collection} = active;
 				const {index} = node.sortableInfo;
 				const margin = getElementMargin(node);
 
 				const containerBoundingRect = this.container.getBoundingClientRect();
-				const dimensions = getHelperDimensions({index, node, collection})
 
 				this.node = node;
-				this.width = dimensions.width
-				this.height = dimensions.height
 				this.margin = margin;
 				this.width = node.offsetWidth;
 				this.height = node.offsetHeight;
```

Comment:
If you look closely here, you can see that the values retrieved from `getHelperDimensions` are set to `this.height` and `this.width` and are overwritten only one line later with `node.offsetWidth` and `node.offsetHeight`. This means that `getHelperDimensions` was never used even though the method existed. 

Most probably this is not desirable but should be resolved in another PR. In this PR however I removed the double assignment, to avoid confusion.